justfile

```diff
@@ -0,0 +1,18 @@
+# format terraform files
+format:
+    terraform fmt -recursive
+
+# lint files
+lint:
+    #!/usr/bin/env bash
+    set -e
+
+    terraform fmt -recursive --check
+    terraform init
+    terraform validate
+    tflint -f compact --recursive
+
+# update autogenerated documentation
+# WARNING: some resource links are bad and need manual fixup :/
+docs:
+    terraform-docs markdown . --output-file README.md
```

Comment:
please add a .terraform.docs.yml file like https://github.com/stacklet/terraform-aws-onboarding-org-read/blob/main/.terraform-docs.yml, ad which point this can be just `terraform-docs .`